sci-libs/rocBLAS/metadata.xml

```diff
@@ -8,4 +8,8 @@
     <email>gentoo@holzke.net</email>
     <name>Wilfried Holzke</name>
   </maintainer>
+    <use>
+        <flag name="tests">Build and install rocblas-test to compare the result between rocBLAS and system BLAS.</flag>
+        <flag name="benchmarks">Build and install rocblas-bench.</flag>
+    </use>
```

Comment:
I prefer `benchmark`.  In ::gentoo, I see 2 packages with `USE=benchmark` and 1 package with `benchmarks`.